WcaOnRails/app/models/user.rb

```diff
@@ -65,7 +65,7 @@ def wca_id_is_unique_or_for_dummy_account
       user = User.find_by_wca_id(wca_id)
       # If there is a non dummy user with this WCA ID, fail validation.
       if user && !user.dummy_account?
-        errors.add(:wca_id, I18n.t('users.errors.unique'))
+        errors.add(:wca_id, I18n.t('users.errors.unique', email: user.email))
```

Comment:
@rwcinoto in order to spare you some time, if you change the name of the key there you will have to change the name in the French locale too [here](https://github.com/rwcinoto/worldcubeassociation.org/blob/4040017b5b9b4f4dd7eb509237c6c7e2efee4bd9/WcaOnRails/config/locales/fr.yml#L385). (otherwise the tests will fail)